src/System.Runtime.Caching/src/MatchingRefApiCompatBaseline.txt

```diff
@@ -0,0 +1,6 @@
+Compat issues with assembly System.Runtime.Caching:
+TypesMustExist : Type 'System.Runtime.Caching.Configuration.CachingSectionGroup' does not exist in the implementation but it does exist in the contract.
```

Comment:
@kkhurin can you confirm whether or not these APIs should be in the reference assembly or whether or not they should be removed from the implementation?

Reply:
Those are the types used for the config, but they are not supposed to be used by the end user. Therefore I removed them from the ref assembly. 
@JeremyKuhne, @danmosemsft, do you guys see any useful case when they would be needed in the ref assembly? How do other libraries who rely on config handle this?

Reply:
Are we actually depending on the configuration system? If not I would just remove these types if possible, otherwise we consider making them internal in the implementation or exposing them publicly in the reference assembly.

Reply:
Yes, we still support configuring cache via config files. 
Don't we need to keep them public in the implementation assembly for the config to work properly?